test/txsim/run_test.go

```diff
@@ -152,3 +157,50 @@ func Setup(t testing.TB) (keyring.Keyring, string, string) {
 
 	return cctx.Keyring, rpcAddr, grpcAddr
 }
+
+func TestTxSimUpgrade(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestTxSimUpgrade in short mode.")
+	}
+	cp := app.DefaultConsensusParams()
+	cp.Version.AppVersion = v2.Version
+	cfg := testnode.DefaultConfig().
+		WithTimeoutCommit(300 * time.Millisecond).
+		WithConsensusParams(cp).
+		WithFundedAccounts("txsim-master")
+	cctx, _, grpcAddr := testnode.NewNetwork(t, cfg)
+
+	// updrade to v3 at height 20
+	sequences := []txsim.Sequence{
+		txsim.NewUpgradeSequence(v3.Version, 20),
+	}
+
+	opts := txsim.DefaultOptions().
+		// SuppressLogs().
```

Comment:
```suggestion
```

Reply:
I uncommented this in another PR I'm working on